app/src/main/res/layout/fragment_activities.xml

```diff
@@ -28,8 +28,15 @@
             android:id="@+id/recycler_view"
             android:layout_width="match_parent"
             android:layout_height="match_parent"
-            tools:layout_editor_absoluteX="130dp"
-            tools:layout_editor_absoluteY="285dp"
-            app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager" />
+            app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"/>
+
+        <TextView
+            android:id="@+id/dog"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="TextView"
```

Comment:
Is it really correct?

Reply:
It is temp, only for displaying dog image url, in next iteration it will be imageView :)